test/fancylogger.py

```diff
@@ -117,7 +123,7 @@ def _reset_fancylogger(self):
 
         # delete all fancyloggers
         loggers = logging.Logger.manager.loggerDict
-        for name, lgr in loggers.items():
+        for name, lgr in list(loggers.items()):
```

Comment:
why `list`?

Reply:
Because `.items()` returns a generator, and we're changing `loggers` while we're looping over it, which means trouble in Python 3